spec/drivers/webkit_spec.rb

```diff
@@ -2,10 +2,13 @@
 require 'shared_examples_for_api'
 require 'capybara-webkit'
 
+Capybara::Webkit.configure do |config|
+  config.allow_url('subdomain.lvh.me')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.